spec/factories/move.rb

```diff
@@ -9,7 +9,6 @@
     time_due { Time.now }
     status { 'requested' }
     additional_information { 'some more info about the move that the supplier might need to know' }
-    move_type { 'court_appearance' }
```

Comment:
There was a balance here - many tests used this as the default move type without specifying the correct from and to location types. Removing this as the default - i.e. move_type is not set and instead derived using the existing `set_move_type` method within `Move` model is less invasive for now. When we move to removing v1 support completely we can revisit this - lots of tests will break again but at least then we only have to worry about half as many request specs.

A few request specs were relying on this assumption so those have been fixed to set an explicit move type and the correct location types (i.e. those won't break in future when `set_move_type` is removed).